test/test_dtype.py

```diff
@@ -35,7 +35,7 @@ def _assert_eq(tensor:Tensor, target_dtype:DType, target):
   if DEBUG >= 2: print(tensor.numpy())
   try:
     assert tensor.dtype == target_dtype
-    np.testing.assert_allclose(tensor.numpy(), target, rtol={dtypes.float16:1e-3, dtypes.bfloat16:1e-2}.get(target_dtype, 1e-7))
+    np.testing.assert_allclose(tensor.numpy(), target, rtol={dtypes.float16:1e-3, dtypes.bfloat16:1e-2}.get(target_dtype, 1e-6))
```

Comment:
why is this needed?

Reply:
there are slight rounding errors due to different order of calculations 
test_dtype.py :: test_arange will produce 5.7999997 instead of 5.8

Reply:
can put it down to 3e-7